### PR TITLE
Added italian quotes

### PR DESCRIPTION
**Pull request by AOTREVAI**
### Description

Closes #

### CODE REVIEW

File: frontend/static/quotes/italian.json [hunk 1]
```diff
@@ -630,6 +630,24 @@
...
+      "text": "Non ho quindici case, le ho solo cambiate.",
+      "source": "Massimo Pericolo, 7 MILIARDI",
+      "length": 42,
+      "id": 106
+    },
+    {
+      "text": "Uomo di panza, uomo di sostanza.",
+      "source": "Saggezza Popolare",
+      "length": 32,
+      "id": 107
```
Comment (Miodec) [inline frontend/static/quotes/italian.json:650]: I think these two are too short. We usually want to keep quotes above 50 characters

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/italian.json [hunk 2]
```diff
@@ -630,6 +630,30 @@
...
+      "text": "Se ni' mondo esistesse un po' di bene e ognun si considerasse suo fratello ci sarebbe meno pensieri e meno pene e il mondo ne sarebbe assai più bello.",
+      "source": "Pietro Pacciani, Processo",
+      "length": 150,
+      "id": 105
+    },
+    {
+      "text": "Noi addestriamo dei giovani a scaricare Napalm sulla gente, ma i loro comandanti non gli permettono di scrivere cazzo sui loro aerei perché è osceno.",
+      "source": "Kurtz, Apocalypse Now",
+      "length": 149,
+      "id": 106
```
Comment (Miodec) [inline frontend/static/quotes/italian.json:644]: Contains mature content, please remove

File: frontend/static/quotes/italian.json [hunk 3]
```diff
@@ -630,6 +630,30 @@
...
+    },
+    {
+      "text": "Ho osservato una lumaca strisciare lungo il filo di un rasoio, questo è il mio sogno, è il mio incubo: strisciare, scivolare lungo il filo di un rasoio e sopravvivere.",
+      "source": "Kurtz, Apocalypse Now",
+      "length": 167,
+      "id": 107
```
Comment (Miodec) [inline frontend/static/quotes/italian.json:650]: References suicide?? Remove

File: frontend/static/quotes/italian.json [hunk 4]
```diff
@@ -630,6 +630,30 @@
...
+    },
+    {
+      "text": "Il napalm, lo senti? Non c'è niente al mondo che abbia questo odore. Mi piace l'odore del napalm al mattino.",
+      "source": "Bill Kilgore, Apocalypse Now",
+      "length": 108,
```
Comment (Miodec) [inline frontend/static/quotes/italian.json:655]: Another quote that pretty much mentions war. Remove

**Review by Miodec — CHANGES_REQUESTED** [review]
Very weird quote selection. Please read the quote guidelines again

File: frontend/static/quotes/italian.json (hunk 4 above)
Comment (AOTREVAI) [reply frontend/static/quotes/italian.json]: it mentions war, but still, it's from a masterpiece movie

**Review by AOTREVAI — COMMENTED** [review]
i removed the inappropriate quotes

**Review by Miodec — APPROVED** [review]

**Pull request merged**